packages/nouns-assets/src/utils.ts

```diff
@@ -82,3 +82,15 @@ export const getNounSeedFromBlockHash = (nounId: BigNumberish, blockHash: string
     glasses: getPseudorandomPart(pseudorandomness, glasses.length, 192),
   };
 };
+
+/**
+ * Get encoded part information for one trait
+ * @param partType The label of the part type to use
+ * @param partIndex The index within the image data array of the part to get
+ */
+ export const getPartData = (partType:string, partIndex:number) => {
+  type ObjectKey = keyof typeof images;
+  const part = partType as ObjectKey;
+  return images[part][partIndex].data;
+};
+exports.getPartData = getPartData;
```

Comment:
`exports.getPartData = getPartData;` is unnecessary

Reply:
resolved in https://github.com/nounsDAO/nouns-monorepo/pull/595/commits/89fcb7e23c0444fb453b48388036134070c513a0